week-3/Homework/mandatory/3-space-colonies.js

```diff
@@ -13,7 +13,13 @@
   
 */
 
-function colonisers() {}
+function colonisers(array) {
+  const lastNames = array.filter(
+    (name) => name.charAt(0) == "A" && name.includes("family")
+  );
+  console.log(lastNames);
```

Comment:
Remember to remove console.log you used for testing!

Reply:
console.log removed.